pom.xml

```diff
@@ -24,14 +24,14 @@
     </organization>
 
     <distributionManagement>
-        <snapshotRepository>
-            <id>ossrh</id>
-            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
-        </snapshotRepository>
         <repository>
-            <id>ossrh</id>
-            <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+            <id>womply</id>
+            <url>https://womply.artifactoryonline.com/womply/libs-release-local</url>
         </repository>
+        <snapshotRepository>
+            <id>womply</id>
+            <url>https://womply.artifactoryonline.com/womply/libs-snapshot-local</url>
```

Comment:
It probably would have been a good idea, but I've since made a second commit based on this one... I think I'll just manually re-craft the commit when doing the PR back